src/containers/App/App.js

```diff
@@ -114,12 +114,12 @@ class App extends React.Component {
     
         <Header />
         
-        <section className="columns is-variable is-0">
-          <div className="column is-narrow is-hidden-mobile">
+        <div className="wrapper">
+          <aside>
```

Comment:
Menu is also an `<aside />`

Reply:
I removed the aside from menu and switched it to a div, making the wrapper an aside.

Reply:
Sounds good! The only issue is that we have a conflict on `<Transfer />`